convert-hf-to-gguf.py

```diff
@@ -2829,19 +2975,22 @@ def main() -> None:
         "auto": gguf.LlamaFileType.GUESSED,
     }
 
-    if args.outfile is not None:
```

Comment:
You removed the optional output file path, but the CLI option still exists. Not sure if this was intentional.

Reply:
It seems to be directly passed to `model_class` below.